Quick-Start/Prerequisite.md

```diff
@@ -60,7 +60,7 @@ PostgreSQL supports various target languages for extension development. Here is
 
 ## C:
 The C programming language is widely used for developing PostgreSQL extensions. It offers direct access to PostgreSQL internals and provides high performance.
- - Development Environment: To develop C-based extensions for PostgreSQL, you need a C compiler (such as GCC) and the necessary development libraries for PostgreSQL. 
+ - Development Environment: To develop C-based extensions for PostgreSQL, you need a C compiler (such as GCC) and the necessary development libraries for PostgreSQL.  
    - On Linux systems, you may need to install the `postgresql-server-dev` package. 
    - On macOS, you can use Xcode Command Line Tools or install PostgreSQL via Homebrew. 
    - On Windows, you can use Microsoft Visual Studio or MinGW.
```

Comment:
Other comments:

* You do not need DBD::Pg to use PL/Perl, but keep the notes about installing stuff from CPAN.
* I suggest you also mention PL/PerlU.
* I don't believe Ruby extensions need `pg` anymore than Perl extensions need DBD::Pg, but you should verify that.
* Same for Java: I don't believe you need the JDBC driver.
*

Reply:
I'll update these immediately, my source material was faulty